protocols/protocol/cloudstate/crdt.proto

```diff
@@ -12,22 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// gRPC interface for Event Sourced Entity user functions.
-
+// gRPC definitions for Crdt Entity user functions.
+//
+//---
+//
+//`Interaction Specification`:
+//
+//  - TODO
 syntax = "proto3";
 
 package cloudstate.crdt;
 
-// Any is used so that domain events defined according to the functions business domain can be embedded inside
-// the protocol.
+// Any is used so that domain events defined according to the functions business domain can be embedded inside the protocol.
 import "google/protobuf/any.proto";
 import "cloudstate/entity.proto";
 
 option java_package = "io.cloudstate.protocol";
 option go_package = "cloudstate/protocol";
 
-// CRDT Protocol
+//The *Crdt* Service is the entrypoint for CRDT-based User Function entities.
```

Comment:
will * Crdt* be the final name?

Reply:
Unlikely, but we haven't settled on a good enough replacement yet.

Reply:
@hepin1989 Follow the discussion here https://github.com/cloudstateio/cloudstate/issues/144